### PR TITLE
Make avro-fastserde depend on helper module instead of avro-migration-helper module

### PR DESCRIPTION
**Pull request by huangminchn**
Also renamed 3 Avro19SchemaNormalization files to SchemaNormalization.

### CODE REVIEW

File: avro-fastserde/build.gradle [hunk 1]
```diff
@@ -31,7 +31,8 @@ if (USE_AVRO_14) {
 }
 
 dependencies {
-  compile project(":avro-migration-helper")
+  compile project(":helper:helper")
+  compile project(":helper:impls:helper-impl-14")
```
Comment (radai-rosenblatt) [inline avro-fastserde/build.gradle]: why is this required?

Comment (huangminchn) [reply avro-fastserde/build.gradle]: Need this function `SchemaNormalization#fingerprint64` from ":helper:impls:helper-impl-14".

Comment (radai-rosenblatt) [reply avro-fastserde/build.gradle]: i think its better to pull up SchemaNormalization into something like com.linkedin.avroutil1.compatibility.SchemaNormalization in the helper module, and consider that API

Comment (huangminchn) [reply avro-fastserde/build.gradle]: Done. Please check the latest commit. Thanks!

Comment (radai-rosenblatt) [reply avro-fastserde/build.gradle]: so is this line still there? github shows it is

Comment (huangminchn) [reply avro-fastserde/build.gradle]: Good catch! This line is not needed any more. Removed.
